src/languages/en.ts

```diff
@@ -508,7 +508,11 @@ export default {
         beginningOfChatHistoryPolicyExpenseChatPartThree: ' starts here! 🎉 This is the place to chat, request money and settle up.',
         beginningOfChatHistorySelfDM: 'This is your personal space. Use it for notes, tasks, drafts, and reminders.',
         chatWithAccountManager: 'Chat with your account manager here',
-        sayHello: 'Say hello!',
+        sayHello: (_count) => ({
```

Comment:
@brandonhenry The lint is failing here. Why this change to the existing `sayHello`, in en.ts is a function, but in es.ts its' a string, why this change?

Reply:
`_count` is not used here, we can remove if not used